src/tests/unit/tests/DetailsView/components/failure-instance-panel-control.test.tsx

```diff
@@ -140,12 +157,28 @@ describe('FailureInstancePanelControlTest', () => {
     });
 
     function createPropsWithType(actionType: CapturedInstanceActionType): FailureInstancePanelControlProps {
+        const featureFlagStoreData = {} as FeatureFlagStoreData;
+        return {
+            step: 'missingHeadings',
+            test: VisualizationType.HeadingsAssessment,
+            addFailureInstance: addInstanceMock.object,
+            actionType: actionType,
+            assessmentsProvider: Assessments,
+            featureFlagStoreData: featureFlagStoreData,
+        };
+    }
+
+    function createPropsWithTypeAndData(
```

Comment:
To minimize duplicated code between this 2 functions, you may want to have a look at our `BaseDataBuilder` (src\tests\unit\common\base-data-builder.ts) which was create for this purposes. ;)